phases/stack.go

```diff
@@ -1,9 +1,98 @@
 package phases
 
-import "fmt"
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/bitrise-io/bitrise/models"
+	"github.com/bitrise-io/go-utils/log"
+	"gopkg.in/yaml.v2"
+)
+
+var defaultStacks = map[string]string{
+	"xamarin":      "osx-vs4mac-stable",
+	"cordova":      "osx-vs4mac-stable",
+	"react-native": "osx-vs4mac-stable",
+	"ionic":        "osx-vs4mac-stable",
+	"flutter":      "osx-vs4mac-stable",
+	"android":      "linux-docker-android",
+	"macos":        "osx-xcode-10.0",
+	"ios":          "osx-xcode-10.0",
+	"other":        "",
+}
+
+var optionsStacks = []string{
```

Comment:
You can use [...] to declare a fixed size array.

Reply:
Indeed, but `[13]string` is a different type than `[]string`, so it's not suitable here.

Reply:
I didn't get this. `[13]string` != `[...]string`. Also I think the cap can be set here.